releasenotes/notes/add-new-mcx-plugins-2177db9195e2b680.yaml

```diff
@@ -0,0 +1,12 @@
+---
+features_transpiler:
+  - |
+    Added multiple high-level-synthesis plugins for synthesizing an :class:`.MCXGate`:
+
+    * :class:`.MCXSynthesis1CleanKG24`, based on :func:`.synth_mcx_1_clean_kg24`.
+    * :class:`.MCXSynthesis1DirtyKG24`, based on :func:`.synth_mcx_1_dirty_kg24`.
+    * :class:`.MCXSynthesis2CleanKG24`, based on :func:`.synth_mcx_2_clean_kg24`.
+    * :class:`.MCXSynthesis2DirtyKG24`, based on :func:`.synth_mcx_2_dirty_kg24`.
+
+    The :class:`.MCXSynthesisDefault` has also been updated to follow the sequence:
+    mcx_2_clean, mcx_2_dirty, mcx_1_clean, mcx_1_dirty when synthesizing MCX gate.
```

Comment:
I'm not sure if this cause the lint/docs error, but I think you should add an extra empty line here.

Reply:
Done